packages/server/src/migrations/schema/v96.ts

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: Copyright Orangebot, Inc. and Medplum contributors
+// SPDX-License-Identifier: Apache-2.0
+/*
+ * This is a generated file
+ * Do not edit manually.
+ */
+
+import { PoolClient } from 'pg';
+import * as fns from '../migrate-functions';
+
+// prettier-ignore
+export async function run(client: PoolClient): Promise<void> {
+  const results: { name: string; durationMs: number }[] = []
+  await fns.query(client, results, `ALTER TABLE IF EXISTS "ServiceRequest" ADD COLUMN IF NOT EXISTS "__reasonCode" UUID[]`);
```

Comment:
Should we consider adding this search param to `DedicatedTokenColumnsOverridesByResourceTypeAndCode` in `packages/server/src/fhir/searchparameter.ts` with a value of `false` so it'll be served by the shared tokens columns rather than getting a dedicated column?

Reply:
I suspect we should since this is likely going to get niche usage.

Reply:
Ah nice, I didn't know that was a thing.  Makes sense to me, I'll change it